src/main/java/seedu/address/logic/commands/CreateMeetingCommand.java

```diff
@@ -67,13 +82,19 @@ private ArrayList<Person> convertNameToPerson(Model model, String[] peopleToMeet
         return output;
     }
 
-    private String peopleToList(ArrayList<Person> arrayOfPeopleToMeet) {
-        String output = "";
+    /**
+     * Converts an ArrayList of Persons to a string of the Persons' names and tags
+     *
+     * @param arrayOfPeopleToMeet ArrayList of Persons to meet in the Meeting
+     * @return a String of all the Persons' names and tags, with every person separated by a line break
+     */
+    public static String peopleToNameAndTagList(ArrayList<Person> arrayOfPeopleToMeet) {
```

Comment:
Quick qn why is there a need to add tags for the display? Shouldn't the name list be enough?

Reply:
I was thinking that displaying tags might lead to greater clarity, especially when the user is meeting people they aren't close with, such that the tags can remind the user which categories those people belong to.